code/obj/sec_tape.dm

```diff
@@ -0,0 +1,82 @@
+/obj/sec_tape
+	anchored = 1
+	icon = 'icons/obj/sec_tape.dmi'
+	icon_state = "sec_tape_s"
+	name = "security cordon"
+	real_name = "security cordon"
+	desc = "A small cordon of security tape, used to keep assistants off crime scenes."
+	density = 1
+	flags = FPRINT | USEDELAY | ON_BORDER
+	object_flags = HAS_DIRECTIONAL_BLOCKING
+	dir = SOUTH
+
+	New()
+		..()
+		layerify()
+		tape_UpdateIcon()
+
+	proc/layerify()
+		SPAWN(3 DECI SECONDS)
+		if (dir == SOUTH)
+			layer = MOB_LAYER + 0.1
+		else
+			layer = OBJ_LAYER
+
+	proc/tape_UpdateIcon()
+		switch (dir)
+			if (WEST)
+				set_icon_state("sec_tape_w")
+			if (NORTH)
+				set_icon_state("sec_tape_n")
+			if (EAST)
+				set_icon_state("sec_tape_e")
+			else //Default position is south
+				return
+
```

Comment:
Add a `set_dir` override somewhere that calls `src.UpdateIcon()`

Reply:
Not entirely sure what you mean since this is inside update_icon()
Should i put this before the override call?

Reply:
I was clicking on the empty line just sorta arbitrarily. Just add a new `set_dir` override somewhere.

Reply:
Done at line 17. Tell me if i messed it up.